rules/windows/builtin/win_susp_proceshacker.yml

```diff
@@ -0,0 +1,24 @@
+title: ProcessHacker Privilege Elevation
+id: c4ff1eac-84ad-44dd-a6fb-d56a92fc43a9
+description: Detects a ProcessHacker tool that elevated privileges to a very high level
+references:
+    - https://twitter.com/1kwpeter/status/1397816101455765504
+author: Florian Roth
+date: 2018/05/27
```

Comment:
Hi a little error in the date, i think it should be 2021/05/27

Reply:
Thanks